doc/source/getting_started/user_guide.rst

```diff
@@ -38,32 +35,27 @@ interpreter from the AEDT installation.
        desktop.add_custom_toolkit("AntennaWizard")
        exit()
 
-#. Close the PyAEDT console.
-
 #. In the AEDT toolbar, click the **AntennaWizard** button to open this wizard in AEDT:
 
-   .. image:: ./_static/toolkit_in_AEDT.png
+   .. image:: ../_static/antenna_catalog.png
       :width: 800
       :alt: Antenna Toolkit in AEDT
 
-   If the toolkit does not open, restart AEDT.
-
-   .. image:: ./_static/design_connected.png
-      :width: 800
-      :alt: UI opened from AEDT, design tab
-
 The Antenna Toolkit Wizard is connected directly to the AEDT session. For wizard usage information,
-see :doc:`toolkit/ui`.
+see :doc:`../toolkit/ui`.
 
 .. _install_toolkit_console_ui:
 
-How to install from the AEDT console and launch as a wizard
------------------------------------------------------------
+Install toolkit from Python console and launch the Antenna wizard
```

Comment:
Proper wizard name? Wizard should probably be capitalized. Is Toolkit in it or not?